src/store/app/index.js

```diff
@@ -52,7 +52,7 @@ export const removeAlert = createAction(
  */
 
 export const getIsMobileDrawerOpened = state => state.app.isMobileDrawerOpened;
-export const getAppWaitingCounter = state => state.app.appWaiting;
+export const getAppWaitingCounter = state => state?.app?.appWaiting || 0;
```

Comment:
The base selectors should not be like this. The base selcotr should return the exact value that in the store without any other functionality, should be pure. If you need some logic then create a new selector for it. Also, if you check the counter logic in the reducer, the counter always has a value, and that can be  less than 0. So return a default one is not a good pattern.

Reply:
I see, I left it as I was trying to solve an issue related to the progress component state. Restored the selector to the previous state